index.html

```diff
@@ -219,12 +219,136 @@ <h2 class="destinations__title">The best destinations</h2>
     <!--SignUp-->
     <section class="sign-up">
       <div class="sign-up__wrap">
-      <img src="./img/user.svg" alt="user" class="sign-up__image" />
-      <p class="sign-up__title">Sign up and get 20% off your first booking</p>
-      <button class="sign-up__btn">Sign up</button>
-      <button class="sign-up__btn-close">
-        <img src="./img/close.svg" alt="X" class="sign-up__image-close" />
-      </button>
+        <img src="./img/user.svg" alt="user" class="sign-up__image" />
+        <p class="sign-up__title">Sign up and get 20% off your first booking</p>
+        <button class="sign-up__btn">Sign up</button>
+        <button class="sign-up__btn-close">
+          <img src="./img/close.svg" alt="X" class="sign-up__image-close" />
+        </button>
+      </div>
+    </section>
+    <!--Reviews-->
+    <section class="reviews">
+      <h2 class="reviews__title">
+        <span class="reviews__title-visible">Guests</span> reviews
+      </h2>
+      <div class="reviews__slider">
+        <ul class="reviews__list">
+          <li class="reviews__item">
+            <div class="reviews__image-wrap">
+              <img
+                src="./img/Resort&Spa.svg"
+                alt="Resort&Spa"
+                class="reviews__image"
+              />
+              <p class="reviews__image-title">From 250 USD</p>
+            </div>
+            <div class="reviews__wrapper">
```

Comment:
It's better to keep html structure by sense, (name of hotel + place) (score + reviews).
It is quite unusual that you decided to save them like this